gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/ProtoMessageRequestFormatter.java

```diff
@@ -141,12 +141,24 @@ public Builder<RequestT> setPath(
       return this;
     }
 
+    @BetaApi
+    public Builder<RequestT> setAdditionalPaths(List<String> rawAdditionalPaths) {
```

Comment:
I think all usages of this method should already be covered by the method below? If that's the case, I would not add this new public method even though it is marked as `BetaAPI`.

Reply:
Gotcha. I'll remove this one and just convert from List<String> to String[].